custom/ilsgateway/filters.py

```diff
@@ -1,23 +1,55 @@
 from django.utils.translation import ugettext_noop
-from corehq.apps.products.models import Product
+from corehq.apps.products.models import SQLProduct
 from corehq.apps.programs.models import Program
 from corehq.apps.reports.filters.base import BaseDrilldownOptionFilter
 
 
 class ProductByProgramFilter(BaseDrilldownOptionFilter):
     slug = "filter_by"
+    single_option_select = 0
+    template = "opm/drilldown_options.html"
```

Comment:
(not blocking) it seems strange for one custom report to depend on another - is there a shared place you can put this?